src/manipulation.js

```diff
@@ -159,6 +159,10 @@ function fixInput( src, dest ) {
 }
 
 jQuery.extend({
+	htmlPrefilter: function( html ) {
```

Comment:
A good question. The dependencies basically look like this:
* `$collection.html( value )`
  * → `jQuery.fn.html` (src/manipulation.js)
    * → **`jQuery.htmlPrefilter`** (src/manipulation.js)
* `$collection[ otherManipulationMethod ]( value )`
  * → `jQuery.fn[ otherManipulationMethod ]` (src/manipulation.js)
    * → `jQuery.fn.domManip` (src/manipulation.js)
      * _(if `value` is a function)_ → `jQuery.fn.html` (src/manipulation.js)
      * → `jQuery.buildFragment` (src/manipulation.js)
        * → **`jQuery.htmlPrefilter`** (src/manipulation.js)
* `jQuery( html )`
  * → `jQuery` (src/core.js)
    * → `jQuery.fn.init` (src/core/init.js)
      * → `jQuery.parseHTML` (src/core/parseHTML.js)
        * → `jQuery.buildFragment` (src/manipulation.js)
          * → **`jQuery.htmlPrefilter`** (src/manipulation.js)
        * _(if `!keepScripts`)_ → `jQuery.fn.remove` (src/manipulation.js)
          * → … (src/traversing/findFilter.js, src/manipulation.js, src/data.js, …)

`htmlPrefilter` is meaningful only for the "manipulation" module, and should not appear in a hypothetical build excluding it. This can be revisited when `buildFragment` is made private, at which point the decision may be to put it in a new module providing `jQuery.htmlPrefilter` and `jQuery( html )`—upon which "manipulation" would depend.